obspy/taup/tau.py

```diff
@@ -4,6 +4,12 @@
                         unicode_literals)
 from future.builtins import *  # NOQA
 
+import copy
+
+import matplotlib.cbook
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
Is there any reason why matplotlib is before scipy? Chronologically plotting comes after the analysis ;-)